python/runner/runner.py

```diff
@@ -186,11 +187,16 @@ async def run_instance(self, config, args):
 
         self.logger.info(f'Sending PANG')
         monitoring = self.streams[CC.MONITORING]
-        pang_provides_data = {
+        pang_data = {
             'provides': '',
             'contentType': ''
         }
-        send_encoded_msg(monitoring, msg_codes.PANG, pang_provides_data)
+        send_encoded_msg(monitoring, msg_codes.PANG, pang_data)
+
+        if hasattr(self.sequence, 'METADATA'):
+            metadata = self.sequence.METADATA
```

Comment:
I'm a bit iffy on this - I'd rather see `produces` and `consumes`... Metadata seems to mean nothing specific.